diff --git gsl_dist/gsl_Extension.py gsl_dist/gsl_Extension.py
index 51c9590..e6fee24 100644
--- gsl_dist/gsl_Extension.py
+++ gsl_dist/gsl_Extension.py
@@ -116,8 +116,8 @@ class gsl_Extension(Extension):
 
 	    if undef_macros == None:
 		    undef_macros = []
-	    if 'NDEBUG' not in undef_macros:
-		    undef_macros.append('NDEBUG')
+	    #if 'NDEBUG' not in undef_macros:
+	    #  undef_macros.append('NDEBUG')
 	    tmp = map(lambda x: x[0], define_macros)
 
 	    # Now config will test of GSL_MAJOR_VERSION and GSL_MINOR_VERSION is defined 
@@ -140,7 +140,7 @@ class gsl_Extension(Extension):
 			       undef_macros,
 			       library_dirs,
 			       libraries,
-			       runtime_library_dirs,
+			       None,
 			       extra_objects,
 			       extra_compile_args,
 			       extra_link_args,
