diff --git Include/pygsl/utils.h Include/pygsl/utils.h
index 98e6ebe..26365ae 100644
--- Include/pygsl/utils.h
+++ Include/pygsl/utils.h
@@ -42,16 +42,16 @@ __BEGIN_DECLS
        :                                                            \
          0  
 
-#define DEBUG_MESS(level, mess, args...)                                 \
+#define DEBUG_MESS(level, mess, args, ...)                            \
      ((PyGSL_DEBUG_LEVEL()) > level)  ?                              \
 fprintf(stderr,                                                      \
 	"In Function %s from File %s at line %d "  mess      "\n" ,  \
-        __FUNCTION__, __FILE__, __LINE__, args)               \
+        __FUNCTION__, __FILE__, __LINE__, args, __VA_ARGS__)                     \
 : \
       0 
 #else /* DEBUG > 0 */
 #define FUNC_MESS(mess)
-#define DEBUG_MESS(level, mess,  args...)
+#define DEBUG_MESS(level, mess,  args, ...)
 #endif /* DEBUG */
 
 #define FUNC_MESS_BEGIN()  FUNC_MESS("BEGIN ")
